assignments/objects.js

```diff
@@ -19,26 +19,68 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+	"id": 1,
+	"name": "Mitzi",
+  	"email": "mmelloy0@psu.edu",
+  	"gender": "F"
+}
+const kennan = {
+	"id": 2,
+	"name": "Kennan",
+  	"email": "kdiben1@tinypic.com",
+  	"gender": "M",
+  	"speak": function(){
+  		return("Hello, my name is Kennan!");
```

Comment:
You don't have to use the parentheses for the [`return`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return) keyword:
```javascript
return "Hello, my name is Kennan!";
```